vcr/assets/assets/default.index.yaml

```diff
@@ -0,0 +1,18 @@
+indices:
+  - name: index_id
+    parts:
+      - iriPath: []
```

Comment:
why's this empty?

Reply:
The root element has no path. In the root it looks for @id.